tests/utils/test_skiperror.py

```diff
@@ -0,0 +1,29 @@
+import pandas as pd
+from janitor.utils import skiperror
+import numpy as np
+
+import pytest
+
+
+@pytest.mark.functions
+def test_skiperror():
+    df = pd.DataFrame({"x": [1, 2, 3, "a"], "y": [1, 2, 3, "b"]})
+    func = lambda s: s + 1
```

Comment:
I like the use of lambda! But to keep the codebase beginner-friendly, can we instead use an internal function definition? i.e.:

```python
def test_skiperror():
...
    def func(s):
        return s + 1
```